analyzer-comments/java/need-for-speed/avoid_loops.md

```diff
@@ -0,0 +1,3 @@
+# avoid loops
+
+Consider converting the `loop` statement and returning a single expression using the `<=` operator while reusing methods already implemented in the class.

```

Comment:
This is quite a long sentence with is a bit hard to read. I'd break it up a bit, something like

```suggestion
Instead of using a loop, consider returning a single expression using the `<=` operator.
You can re-use the other methods already implemented in the class.
```